frame/staking/src/lib.rs

```diff
@@ -1396,6 +1402,14 @@ decl_module! {
 		}
 
 		fn integrity_test() {
+			// Ensure the size of both ValidatorIndex and NominatorIndex. They both need to be well
+			// below usize.
```

Comment:
sidenote:

```suggestion
			// Ensure the size of both ValidatorIndex and NominatorIndex. They both need to be less or
			// equal to usize.
```
Also it would make more sense to compare their max_value. But I'm fine here